app/models/user.rb

```diff
@@ -74,7 +74,9 @@ def self.for_ichain_username(username, attributes)
     raise UserDisabled if user && user.is_disabled
 
     if user
-      user.update_attributes(email: attributes[:email])
+      user.update_attributes(email: attributes[:email],
+                             last_sign_in_at: user.current_sign_in_at,
+                             current_sign_in_at: Time.current)
```

Comment:
Obviously not :(
Your last login was somewhere 2014 and I have no last login set at all ...